### PR TITLE
EPMRPP-62670: Upload a .jar file and view new tabs

### PR DESCRIPTION
**Pull request by Alexandr-Kislov**
https://jiraeu.epam.com/browse/EPMRPP-62670

DESCRIPTION:
1) Added plugin-example project
2) Made some changes to define the plugin

### CODE REVIEW

**Comment by VyacheslavGarshin** [comment]
description of PR shouldn't repeat the task itself=)
it's better to describe what exactly in the MR.
example: 1) added some projects 2) created some sql scripts 3) added several functions... etc.
several main changes would be nice!

**Comment by Alexandr-Kislov** [comment]
> description of PR shouldn't repeat the task itself=)
> it's better to describe what exactly in the MR.
> example: 1) added some projects 2) created some sql scripts 3) added several functions... etc.
> several main changes would be nice!

Changed the description of the PR

File: src/main/resources/schema/001_schema_init.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+CREATE SCHEMA IF NOT EXISTS example;
```
[the remark this answers is not in the document]
Comment (Alexandr-Kislov) [reply src/main/resources/schema/001_schema_init.sql:1]: Changed it in the scope of this [PR](https://github.com/reportportal/plugin-bts-azure/pull/3)

File: src/main/java/com/epam/reportportal/extension/example/ExampleExtension.java [hunk 2]
```diff
@@ -0,0 +1,186 @@
+package com.epam.reportportal.extension.example;
```
[the remark this answers is not in the document]
Comment (Alexandr-Kislov) [reply src/main/java/com/epam/reportportal/extension/example/ExampleExtension.java:1]: Changed it in the scope of this [PR](https://github.com/reportportal/plugin-bts-azure/pull/3)

File: src/main/resources/schema/001_schema_init.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+CREATE TABLE IF NOT EXISTS example.entity
```
[the remark this answers is not in the document]
Comment (Alexandr-Kislov) [reply src/main/resources/schema/001_schema_init.sql:3]: All the basic entities we save in the core schema. This entity is just an example for a situation if we will need some specific entities for our plugin, which we will save in our schema.
If it doesn't come in handy in the future I'll just delete it.

**Pull request merged**